src/event/model/event.rs

```diff
@@ -91,9 +110,19 @@ impl Event {
             "entity/link/remove" => Event::RemoveEntityLink(RemoveEntityLink::build(e)),
             "entity/revision/add" => Event::CreateEntityRevision(CreateEntityRevision::build(e)),
             "entity/revision/checkout" => {
-                let repository_id = fetch_parameter_uuid_id(e.id, "repository", &pool).await;
+                let repository_id = fetch_parameter_uuid_id(e.id, "repository", &pool)
+                    .await
+                    .unwrap();
                 let reason = fetch_parameter_string(e.id, "reason", &pool).await;
-                Event::CheckoutRevision(CheckoutRevision::build(e, repository_id.unwrap(), reason))
+                let revision_id = abstract_event.object_id;
+                Event::CheckoutRevision(CheckoutRevision {
+                    abstract_event,
+
+                    __typename: String::from("CheckoutRevisionNotificationEvent"),
```

Comment:
Would probably be even cleaner using Valentin's pattern:
- Enum https://github.com/serlo/serlo.org-database-layer/pull/23/files#diff-22cee5576a81bdb8d679e1d0f63bf4ef19552e831836d80ecf548213c881883cR47
- Parse from string https://github.com/serlo/serlo.org-database-layer/pull/23/files#diff-b6f1f849ff97d9bdafb07928931bdd1404acf4b0730e8a39ed8efbd75c52f3eaR32

Reply:
This would also allow to move the `__typename` into `AbstractEvent`.